tests/deck/test_deck.py

```diff
@@ -69,6 +69,14 @@ def test_find_flushes(self) -> None:
 
         self.assertEqual(f"{expected}", f"{actual}")
 
+    def test_is_a_flush(self) -> None:
+        deck = F.new_deck()
+        hand = deck.cut_cards(5)
```

Comment:
shall we mock the hand instead of using a method to define it?

Reply:
I think this is fine, maybe using `F.clubs_in_order()` as the deck would be clearer, a test for `is_a_flush_false` would be good too.
```
deck = F.from_string("AC 2C 3C 4C 5D 6D 7H")
hand = deck.cut_cards(5)

self.assertFalse(hand.is_a_flush())
```

When the result is a `boolean` prefer using `self.assertTrue(hand.is_a_flush())` as the lisibility is already good